### PR TITLE
Tests: Tests refactoring for min nb journeys

### PR DESCRIPTION
**Pull request by benoit-bst**
This PR contains :

- Suppress old tests because the kraken mock doesn't contain a good data example
- Create a bunch of new tests for min_nb_journeys  (With a new mock)
- Create a bunch of new tests for nigth bus filter  (With a new mock)
- Fix Night bus filter parameters for distributed

Kisses 
:heart: 

JIRA : https://jira.kisio.org/browse/NAVP-903

### CODE REVIEW

File: source/jormungandr/tests/journey_common_tests.py [hunk 1]
```diff
@@ -1487,3 +1451,113 @@ def test_min_nb_transfers(self):
...
+
+@dataset({"min_nb_journeys_test": {}})
+class JourneysMinNbJourneys():
+    """
+    Test min_nb_journeys and late journey filter
+    """
+
+    def test_minimum_journeys_with_min_nb_journeys_options(self):
+        """
+        By default, the raptor compute 2 journeys, so the response return to the minimum 2 journeys.
```
Comment (mbounabi) [inline source/jormungandr/tests/journey_common_tests.py]: *compute**s**
*renturn**s** **at least**

File: source/jormungandr/tests/journey_common_tests.py [hunk 2]
```diff
@@ -1487,3 +1451,113 @@ def test_min_nb_transfers(self):
...
+
+        Note : The night bus filter is loaded with default parameters.
+        With this data, night bus filter parameters don't filter anything.
```
Comment (mbounabi) [inline source/jormungandr/tests/journey_common_tests.py]: *doesn't

File: source/jormungandr/tests/journey_common_tests.py [hunk 3]
```diff
@@ -1487,3 +1451,113 @@ def test_min_nb_transfers(self):
...
+        """
+        query = "journeys?from=2.39592;48.84838&to=2.36381;48.86750&datetime=20180309T080000&min_nb_journeys=3"
+        response = self.query_region(query)
+        self.is_valid_journey_response(response, query)
+        assert len(response['journeys']) == 3
+
+    def test_maximum_journeys_with_min_nb_journeys_options(self):
+        """
+        The data contains only 5 journeys but we want 6 journeys.
+        The response have to contains only 5 journeys.
```
Comment (mbounabi) [inline source/jormungandr/tests/journey_common_tests.py]: ha**s** to contain (no 's')

File: source/jormungandr/tests/journey_common_tests.py [hunk 4]
```diff
@@ -1487,3 +1451,113 @@ def test_min_nb_transfers(self):
...
+        With this data, night bus filter parameters don't filter anything.
+        """
+        query = "journeys?from=2.39592;48.84838&to=2.36381;48.86750&datetime=20180309T080000&min_nb_journeys=1"
+        response = self.query_region(query)
+        self.is_valid_journey_response(response, query)
+        assert len(response['journeys']) == 2
+
+    def test_min_nb_journeys_options(self):
+        """
+        The data contains only 5 journeys, so the response return to the minimum 3 journeys.
```
Comment (mbounabi) [inline source/jormungandr/tests/journey_common_tests.py]: Same comment as above

File: source/jormungandr/tests/journey_common_tests.py [hunk 5]
```diff
@@ -1487,3 +1451,113 @@ def test_min_nb_transfers(self):
...
+
+
+@dataset({"min_nb_journeys_test": {}})
+class JourneysWithNightBusFilter():
+
+    def classical_night_bus_filter_parameters(self):
+        """
+        The data contains 5 journeys with min_nb_journeys = 5.
+        We active the min_nb_journeys option and change night bus filter parameters
+        (ax + b with a = 1.2, b = 0) to filter the 2 lastest journeys.
```
Comment (mbounabi) [inline source/jormungandr/tests/journey_common_tests.py]: *latest

**Review by mbounabi — APPROVED** [review]

File: source/jormungandr/tests/journey_common_tests.py (hunk 1 above)
Comment (benoit-bst) [reply source/jormungandr/tests/journey_common_tests.py]: :white_check_mark:

File: source/jormungandr/tests/journey_common_tests.py (hunk 2 above)
Comment (benoit-bst) [reply source/jormungandr/tests/journey_common_tests.py]: :white_check_mark:

File: source/jormungandr/tests/journey_common_tests.py (hunk 3 above)
Comment (benoit-bst) [reply source/jormungandr/tests/journey_common_tests.py]: :white_check_mark:

File: source/jormungandr/tests/journey_common_tests.py (hunk 4 above)
Comment (benoit-bst) [reply source/jormungandr/tests/journey_common_tests.py]: :white_check_mark:

File: source/jormungandr/tests/journey_common_tests.py (hunk 5 above)
Comment (benoit-bst) [reply source/jormungandr/tests/journey_common_tests.py]: :white_check_mark:

**Review by nonifier — APPROVED** [review]
:+1:

File: source/jormungandr/tests/routing_tests_new_default.py [hunk 6]
```diff
@@ -405,3 +405,11 @@ class TestNewDefaultOneDeadRegion(OneDeadRegion, NewDefaultScenarioAbstractTestF
...
 class TestNewDefaultWithoutPt(WithoutPt, NewDefaultScenarioAbstractTestFixture):
     pass
+
+@config({"scenario": "new_default"})
+class TestNewDefaultMinNbJourneys(JourneysMinNbJourneys, NewDefaultScenarioAbstractTestFixture):
+    pass
+
+@config({"scenario": "new_default"})
+class TestNewDefaultWithNightBusFilter(JourneysWithNightBusFilter, NewDefaultScenarioAbstractTestFixture):
+    pass
```
Comment (kadhikari) [inline source/jormungandr/tests/routing_tests_new_default.py:415]: Are these two test classes **JourneysMinNbJourneys** and **JourneysWithNightBusFilter** not supported in new_default and distributed or deactivated temporarily?

**Review by kadhikari — APPROVED** [review]

File: source/jormungandr/tests/routing_tests_new_default.py (hunk 6 above)
Comment (benoit-bst) [reply source/jormungandr/tests/routing_tests_new_default.py:415]: I have created two differents classes to segment tests, *JourneysMinNbJourneys* and *JourneysWithNightBusFilter*. 
They are called in **new_default** and **distributed**.

File: source/jormungandr/tests/journey_common_tests.py [hunk 7]
```diff
@@ -197,42 +197,6 @@ def test_datetime_represents_arrival(self):
         self.is_valid_journey_response(response, query)
         assert len(response["journeys"]) >= 3
 
-    def test_min_nb_journeys(self):
```
Comment (xlqian) [inline source/jormungandr/tests/journey_common_tests.py:200]: :+1:

**Review by xlqian — APPROVED** [review]

**Pull request merged**